.github/workflows/build.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java: [ 11, 13, 15, 16 ]
```

Comment:
Please, keep 13 and 15 since we are not winning anything by removing those.

Reply:
We are ... in the sense we are removing some flakiness in the test suite ... not necessarily the tests themselves ...